server/db/queries/user.ts

```
@@ -1,49 +1,87 @@
+//     id: admin.id,
+//     phoneNumber: admin.phoneNumber,
+//     department: admin.department,
+//     accessLevel: admin.accessLevel,
+//     hostelId: admin.hostelId,
+//     status: admin.status,
+//   },
+// };
+
+const today = new Date().toISOString();
 
 const today = new Date().toISOString();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'today'. Consider to delete it or rename it.

'today' is defined here:


(lint/suspicious/noRedeclare)